<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pathfinding Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="BtnConfirm">
     <property name="text">
      <string>Map Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnPosConfirm">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Ready for action</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnAction">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start!</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
